src/app/components/dashboard-apps-dock/no-apps-notification.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
The template elements need to be wrapped to show, otherwise it will not display what is left outside. So with not dive wrapper, the bold text won't be rendered.

  <div class="close top-right-corner" type="button" aria-label="Close" ng-click='toggle()'>x</div>
  <strong><i class='fa fa-info-circle' aria-hidden='true'></i></strong>
  {{ 'mno_enterprise.templates.impac.dock.do_not_have_apps' | translate }} 
  <a ui-sref='onboarding.step2'>{{ 'mno_enterprise.templates.impac.dock.click_me' | translate }} </a> **{{ 'mno_enterprise.templates.impac.dock.see_onboarding' | translate }}**